src/calculateRentalCost.js

```diff
@@ -4,7 +4,7 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  console.log(days);
```

Comment:
The `console.log` statement on line 7 should be removed as it is used for debugging purposes and should not be present in the final implementation.